### PR TITLE
Fix Global donation link

### PR DESCRIPTION
**Pull request by zackkrida**
## Fixes

Related to https://github.com/creativecommons/tech-support/issues/716 by @annatuma 

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x]  I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: docs/src/components/Footer.vue [hunk 1]
```diff
@@ -47,9 +81,27 @@
...
+                      href="https://github.com/creativecommons/vocabulary/releases"
+                      class="menu-item"
+                      rel="noopener"
+                      >Release Notes</a
+                    >
+                  </li>
+                  <li>
+                    <a href="https://cc-vocabulary.netlify.app" class="menu-item" rel="noopener"
+                      >Documentation</a
+                    >
```
Comment (dhruvkb) [inline docs/src/components/Footer.vue:95]: Is this the preferred way of indentation according to Prettier? That's not "prettier" 😄.

File: docs/src/components/Footer.vue [hunk 2]
```diff
@@ -84,8 +148,14 @@
               <aside class="donate-section">
                 <h5>Our work relies on you!</h5>
                 <p>Help us keep the internet free and open.</p>
-                <a class="button small donate" href="http://creativecommons.org/donate" rel="noopener">
-                  <i class="icon cc-letterheart-filled margin-right-small is-size-5 padding-top-smaller"></i>
+                <a
+                  class="button small donate"
+                  href="https://creativecommons.org/donate?c_src=website&c_src2=GlobalFooter"
+                  rel="noopener"
+                >
```
Comment (dhruvkb) [inline docs/src/components/Footer.vue:155]: Was very hard to find the actual change in the file. LGTM.

**Review by dhruvkb — COMMENTED** [review]
There are quite a few linting errors as pointed out by GitHub Actions, LGTM otherwise.

**Comment by zackkrida** [comment]
Thanks for reviewing @dhruvkb. I'll talk to Olga today about finally sorting out prettier and linting for vocabulary.

**Pull request merged**